### PR TITLE
#167749946 Setup Swagger

### PR DESCRIPTION
**Pull request by davidteejay**
### What does this PR do?
Sets up Swagger Documentation for the project

### Description of Task to be completed?
Created the `swagger.json` file and set up the `/docs` route for viewing the documentation

### How should this be manually tested?
To test:
- Visit the /docs routes in a browser

### What are the relevant pivotal tracker stories?
[#167749946](https://www.pivotaltracker.com/story/show/167749946)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
+    "swagger-jsdoc": "^3.4.0",
```
Comment (hustlaviola) [inline package.json]: This package seems redundant, you might want to remove it

**Review by hustlaviola — CHANGES_REQUESTED** [review]
Awesome. This is ready to go except for the es5 syntax. Please fix that. Good job

**Comment by davidteejay** [comment]
The develop branch is in es5 syntax

**Review by hustlaviola — APPROVED** [review]
Good job

**Review by ascii-dev — APPROVED** [review]
LGTM

**Pull request merged**